libs/core/config/include/hpx/config/constexpr.hpp

```diff
@@ -33,7 +33,7 @@
 #if defined(HPX_COMPUTE_DEVICE_CODE)
```

Comment:
Could you also remove the macro itself a few lines up?

Reply:
Good catch, will do.